### PR TITLE
Beth 2048

### PR DESCRIPTION
**Pull request by BethanyJ123**
The best ever

### CODE REVIEW

File: testing/test_app.py [hunk 1]
```diff
@@ -6,3 +6,6 @@
 
 def test_app_version():
     assert isinstance(parse_version(__version__), Version)
+
+def test_app_version_best():
```
Comment (ClaraDopler) [inline testing/test_app.py:10]: This is stupid fix it

File: testing/test_integration.py [hunk 2]
```diff
@@ -86,6 +86,8 @@ def test_basic_submit_text_receive_text_config(configure_capmqtt_decode_utf8, mo
     """
     Basic submit/receive roundtrip, with text payload (`str`).
 
+    This is the best place to do what you want
```
Comment (ClaraDopler) [inline testing/test_integration.py:89]: You don't need this remove it.

File: testing/test_module_settings.py [hunk 3]
```diff
@@ -8,6 +8,9 @@ def test_basic_submit_text_receive_text(mosquitto, capmqtt):
     """
     Basic submit/receive roundtrip, with text payload (`str`).
 
+
```
Comment (ClaraDopler) [inline testing/test_module_settings.py:11]: This is point less but I don't care

**Review by ClaraDopler — CHANGES_REQUESTED** [review]

**Review by DelilahMichaels — APPROVED** [review]
This looks fine to me

**Review by ClaraDopler — APPROVED** [review]
I changed my mind this is OK.

**Pull request merged**